website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
OK, so perhaps we go with @brianjgeiger's suggestion:

```
/project/{pid_of_pending_registration}/?approval={token} 
/project/{pid_of_pending_registration}/?disapproval={token}
```

Good to go, @brianjgeiger @JeffSpies ?